src/ovp_pipeline/layer_schemas.py

```
@@ -0,0 +1,570 @@
+    if value != EVERGREEN_ENTITY_TYPE:
+        return f"evergreen entity_type must be {EVERGREEN_ENTITY_TYPE!r}, got {value!r}"
+    return None
+
+
+def _validate_entity_type_in_layer_types(value: Any, _file: Path) -> str | None:
+    if value not in ENTITY_LAYER_TYPES:
+        return f"entity_type {value!r} not in {sorted(ENTITY_LAYER_TYPES)}"
+    return None
+
+
+def _validate_evergreen_tag_present(value: Any, _file: Path) -> str | None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)